doc/api/buffer.markdown

```diff
@@ -75,8 +75,8 @@ The character encodings currently supported by Node.js include:
 Buffers are also `Uint8Array` TypedArray instances. However, there are subtle
 incompatibilities with the TypedArray specification in ECMAScript 2015. For
 instance, while `ArrayBuffer#slice()` creates a copy of the slice,
-the implementation of [`Buffer#slice()`][] creates a view over the existing
-Buffer without copying, making `Buffer#slice()` far more efficient.
+the implementation of [`Buffer#slice()`][`buf.slice()`] creates a view over the
```

Comment:
Well, since it points to `buf.slice()` reference. I could do something like [buf-slice] and use it in both cases. But this way I guess I saved some characters ))